cli_ftd/user/sensor/stm32/stm32_i2c.c

```diff
@@ -1,5 +1,13 @@
 #include "sensor/stm32/stm32_i2c.h"
 
+/* Standard Library Header files */
+#include <assert.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/* POSIX Header files */
+#include <unistd.h>
```

Comment:
unistd is for sleep, this isnt used here

Reply:
Done. Updated in new PR